lib/utils/platform/iot_clock_posix.c

```diff
@@ -163,7 +163,7 @@ bool IotClock_GetTimestring( char * const pBuffer,
     bool status = true;
     const time_t unixTime = time( NULL );
     struct tm localTime = { 0 };
-    size_t timestringLength = 0;
+    int timestringLength = 0;
```

Comment:
Is it good to use int here, instead of fixed size?

Reply:
Good catch! I should not have changed this one. In a place like this you do not really get to choose the data type, if this variable is assigned from strftime then the type should be the same as the function. I was intending to make the change which Gordon pointed out to this function in iot_clock_afr.c where it was using strnprintf, which returns int, that is why it had to be int, but strftime returns size_t, which means this must remain size_t. I will fix this by changing it back.